downloads/swan-lake-release-notes/swan-lake-2201.3.0/RELEASE_NOTE.md

```diff
@@ -125,6 +125,8 @@ Previously, `-9223372036854775808` was allowed to be used as an integer in Balle
 int result = -9223372036854775808; // error: '9223372036854775808' is out of range for 'int'
 ```
 
+Note that this change has affected the UUID module. Therefore, if an application uses the UUID module or uses a package which has a dependency on the UUID module, make sure that the UUID version is updated to 1.4.0. Otherwise, it would result in the aforementioned compilation error.
```

Comment:
```suggestion
>**Note:** This change has affected the `uuid` module. Therefore, if an application uses the`uuid` module or uses a package, which has a dependency on the `uuid` module, make sure that the UUID version is updated to 1.4.0. Otherwise, it would result in the aforementioned compilation error.
```